waiter/src/waiter/metrics.clj

```diff
@@ -93,6 +94,11 @@
   [classifier & nested-path]
   `(counters/counter ~(metric-name (concat ["waiter" classifier "counters"] nested-path))))
 
+(defmacro waiter-gauge
```

Comment:
Using gauges is obviously new.  We avoided guages in the past, but unsure if we need to continue to avoid them.